pkg/manifestgen/sourcesecret/options.go

```diff
@@ -54,12 +60,18 @@ type Options struct {
 	Keypair             *ssh.KeyPair
 	Username            string
 	Password            string
-	CAFile              []byte
-	CertFile            []byte
-	KeyFile             []byte
+	CACrt               []byte
+	TlsCrt              []byte
+	TlsKey              []byte
 	TargetPath          string
 	ManifestFile        string
 	BearerToken         string
+
+	// Depreacted: These fields are used to store TLS data that
+	// specified by the deprecated TLS flags.
+	CAFile   []byte
+	CertFile []byte
+	KeyFile  []byte
```

Comment:
```suggestion
	// Deprecated: Replaced by CACrt, but kept for backwards compatibility 
	// with deprecated TLS flags.
	CAFile   []byte
	// Deprecated: Replaced by TLSCrt, but kept for backwards compatibility
	// with deprecated TLS flags. 
	CertFile []byte
	// Deprecated: Replaced by TLSKey, but kept for backwards compatibility
	// with deprecated TLS flags.
	KeyFile  []byte
```

To ensure highlighting in IDEs per field, including information about what should be used instead.